pages/index.js

```diff
@@ -49,32 +86,73 @@ export default function ShoppingItemsList({
             mode={mode}
           />
         )}
-
-        {mode !== "add" && (
-          <AddItemContainer>
-            <>
-              <p>...need anything else?</p>
+        {mode !== "add" ||
+          (!mode && (
+            <AddItemContainer>
+              <>
+                <p>...need anything else?</p>
+              </>
               <AddButton onClick={() => handleChangeMode("add")}>+</AddButton>
-            </>
-          </AddItemContainer>
+            </AddItemContainer>
+          ))}
+        {unpurchasedItems.length !== 0 && (
```

Comment:
To keep it consistent with the `purchasedItems` list, I suggest to adapt this to:

```suggestion
        {unpurchasedItems.length > 0 && (
```

Reply:
Done.